package.json

```diff
@@ -6,13 +6,12 @@
   "types": "./types.d.ts",
   "scripts": {
     "build": "npm run clean && rollup -c",
-    "clean": "rm -rf ./build",
+    "clean": "rm -rf ./build && rm docs/error.md",
     "docs": "docsify serve docs",
     "lint": "tslint -c tslint.json 'src/**/*.ts'",
     "lint:fix": "tslint -c tslint.json --fix 'src/**/*.ts'",
-    "release": "npm run build; cd build; npm config set '//registry.npmjs.org/:_authToken' $NPM_TOKEN; npm publish",
-    "server": "echo \"Demo is available http://localhost:8080/demo\" && http-server -p 8080",
-    "start": "nodemon --exec \"npm run build && npm run server\" --kill-all --watch ./src -e ts",
+    "release": "npm run clean && npm run build; cp ./package.json dist; cp ./README.md dist; cp src/types.ts dist; cp -R umd dist; cd dist; npm config set '//registry.npmjs.org/:_authToken' $NPM_TOKEN; npm publish",
```

Comment:
Just a heads up, this script runs inside a shell on an AWS build box so there will never be any cached build information. All that to say we dont need this because our deployments are automated. No worries though.